app/frontend/components/pages/delegations/delegatePage.js

```diff
@@ -0,0 +1,58 @@
+const {h} = require('preact')
+const connect = require('unistore/preact').connect
+const actions = require('../../../actions')
+
+const DelegatePage = ({
+  poolAddress,
+  updatePoolAddress,
+  setAdalitePool,
+  submitDelegation,
+  currentStakepool,
+}) => {
+  return h(
+    'div',
+    {class: 'delegate card'},
+    h('h2', {class: 'card-title'}, 'Delegate Stake'),
+    h('input', {
+      type: 'text',
+      id: 'pool-address',
+      class: 'input pool-address fullwidth',
+      name: 'pool-address',
+      placeholder: 'Stakepool Address',
+      value: poolAddress,
+      onInput: updatePoolAddress,
+      autocomplete: 'off',
+    }),
+    h(
+      'button',
+      {
+        class: 'button delegate',
+        id: 'add-pool',
+        onClick: setAdalitePool,
+        disabled: false,
+      },
+      'AdaLite stake pool'
+    ),
```

Comment:
Personally I'd rather have it as an underlined pink text inside the text input, as it is implemented with sending Max funds 
![image](https://user-images.githubusercontent.com/22454337/69374893-66ae6b80-0ca7-11ea-944d-e6628f4b980c.png) but that can be discussed.

Reply:
This is no longer relevant. We will be reverting back to multiple stake pools.